shopify-client/src/views/ShopList.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+  <div class="shopList">
+    <h1>Shop List</h1> <!--Temp header-->
+    <div id="filters">
+      <div class="filterTile">
+        <h4>Categories</h4>
+        <div v-if="categories.length < 1">
+          <p>No available categories</p>
+        </div>
+        <div
+            v-else
+            v-for="category in categories"
+            :key="category.id"
```

Comment:
Can you use `v-bind:<attribute>`. `v-bind` ensures that the attributes will be updated if it changes.

Reply:
sure, others says : is the same thing but I guess they missed that

Reply:
If it is then don't change it.